benefits/core/templates/core/includes/agency-selector.html

```diff
@@ -0,0 +1,31 @@
+{% extends "core/includes/modal.html" %}
+{% load i18n %}
+{% load static %}
+
+{% block modal-content %}
+    <div class="container text-center">
+        <h3 id="{{ modal.aria_labelledby_id }}" class="modal-title">{% translate "core.pages.index.agency_selector.headline" %}</h3>
+        <div class="row gx-lg-5 mx-lg-1 d-flex flex-column flex-lg-row text-start text-lg-center">
+            {% for agency in modal.agencies %}
+                <div class="col">
+                    <div class="card mt-lg-4 mx-lg-auto">
+                        <div class="card-body d-flex flex-row-reverse flex-lg-column justify-content-center align-items-center">
+                            <img class="sm d-lg-none ms-auto"
+                                 src="{% get_static_prefix %}img/agencies/{{ agency.slug }}-sm.svg"
+                                 width="46"
+                                 height="29"
+                                 alt="{% blocktranslate with agency_short_name=agency.short_name %}core.pages.index.agency_selector{{ agency_short_name }}{% endblocktranslate %}">
+                            <img class="lg d-none d-lg-block mt-lg-3 mx-lg-auto"
+                                 src="{% get_static_prefix %}img/agencies/{{ agency.slug }}-lg.svg"
+                                 width="117"
+                                 height="74"
+                                 alt="{% blocktranslate with agency_short_name=agency.short_name %}core.pages.index.agency_selector{{ agency_short_name }}{% endblocktranslate %}">
+                            <h4 class="card-title mt-lg-3 mb-0 mb-lg-3">{{ agency.long_name }}</h4>
+                            <a href="{{ agency.eligibility_index_url }}" class="stretched-link"></a>
```

Comment:
I wonder if there's a way to move this link or add styles to it so this link does not affect the height/flow of the parent div. Also, I wonder if there's a way to add styles so that the focus ring of this link borders the entire div cleanly.

Reply:
Similarly, this link needs to at least wrap the `h4` text, we shouldn't have a link with no text content.

Reply:
FWIW @machikoyasuda I think this `stretched-link` utility is the right way to do this for cards in Bootstrap: https://getbootstrap.com/docs/4.6/utilities/stretched-link/

But we do need text content in the link